tasks/powershell.ps1

```diff
@@ -2,7 +2,14 @@
 
 # Delegate to facter if available
```

Comment:
I'm leaning towards dropping delegating to facter in the powershell and bash scripts at all. It complicates them and makes them less predictable. If you want to use the facter implementation, then add the `puppet-agent` feature on your target and you'll get actual facter.

However this is a substantial change. We should probably get a few more opinions.